pkg/cluster/px/px.go

```diff
@@ -1247,11 +1248,22 @@ func (ops *pxClusterOps) deleteAllPXComponents(clusterName string) error {
 			pureCloudDriveConfigMap,
 			lhConfigMap,
 			storkControllerConfigMap,
+			pxAttachDrivesetConfigMap,
 			// below 2 are currently only in the kube-system namespace. keeping them here
 			// in case the behavior changes
 			fmt.Sprintf("%s%s", internalEtcdConfigMapPrefix, strippedClusterName),
 			fmt.Sprintf("%s%s", cloudDriveConfigMapPrefix, strippedClusterName),
 		}
+
+		// Get per zone cloud drive attach driveset config maps
+		cms, err := ops.coreOps.ListConfigMap(ns, metav1.ListOptions{})
```

Comment:
While this might still work, Attach will not have a zone suffix. Only BringUpQueue does.
nit: prefixConfigMaps = []string{
pxBringupQueueConfigMapPrefix
}
Then have the generic for loop like the one before.
Easy to add more in the future.

Reply:
done